Wikipedia/Code/ScrollableEducationPanelViewController.swift

```diff
@@ -182,7 +182,7 @@ class ScrollableEducationPanelViewController: UIViewController, Themeable {
         dismiss(animated: true, completion: nil)
     }
     
-    @IBAction fileprivate func primaryButtonTapped(_ sender: Any) {
+    @IBAction open func primaryButtonTapped(_ sender: Any) {
```

Comment:
It looks like this class was designed to put actions in the tap handler rather than override the IBAction. 

Either strategy is fine, but now it looks like both are happening. Probably want to settle on one strategy so subclass implementers have an understanding of what is expected